### DISCUSSION TITLE
Age and adoption not being reported correctly in merge confidence report

### DISCUSSION THREAD

**Discussion by sekhavati** [renovatebot/renovate#15328] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

From maybe around a week ago a lot of the PRs opened by Renovate in our repo are reporting Age/Adoption as New/0%. For example below is a screenshot of a well known package, where the PR was opened 4 days ago to bump the version. I'm also now unsure whether the Confidence score is accurately being reported either. 

![Screenshot 2022-04-27 at 09 36 37](https://user-images.githubusercontent.com/16732873/165478269-f9193f21-ed4c-4443-a5c0-cceb6b0e31bd.png)

This is our Renovate config if it matters:

```
{
  "extends": [
    "config:base"
  ],
  "timezone": "Europe/London",
  "schedule": [
    "before 3am on sunday"
  ],
  "packageRules": [
    {
      "matchPackageNames": ["node"],
      "allowedVersions": "14.x"
    },
    {
      "matchPackageNames": ["npm"],
      "allowedVersions": "6.x"
    }
  ]
}
```

### Relevant debug logs

<details><summary>Logs</summary>

The only thing I could see mentioned in the logs about age, adoption or confidence was the snippet below. I noted it was tagged as `beta` (`github>whitesource/merge-confidence:beta`)

```
DEBUG: No config migration necessary
DEBUG: massaged config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "config:base"
    ],
    "timezone": "Europe/London",
    "schedule": [
      "before 3am on sunday"
    ],
    "packageRules": [
      {
        "matchPackageNames": [
          "node"
        ],
        "allowedVersions": "14.x"
      },
      {
        "matchPackageNames": [
          "npm"
        ],
        "allowedVersions": "6.x"
      }
    ]
  }
}
DEBUG: migrated config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "config:base"
    ],
    "timezone": "Europe/London",
    "schedule": [
      "before 3am on sunday"
    ],
    "packageRules": [
      {
        "matchPackageNames": [
          "node"
        ],
        "allowedVersions": "14.x"
      },
      {
        "matchPackageNames": [
          "npm"
        ],
        "allowedVersions": "6.x"
      }
    ]
  }
}
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
This has nothing todo with renovate itself, those data is only computet by hosted app.

Maybe there is an database issue or something else.

Only @rarkins can help there.

**Reply by rarkins** [reply]
I'll take a look, thanks

**Reply by rarkins** [reply]
I think you should see fixed/updated results rolling out over the next few hours, and probably data should be fully complete within a day. Please confirm what you're seeing and thanks for raising it!

**Reply by sekhavati** [reply]
@rarkins yep I can see the figures are now alive again 🙌  

As always thank you for your prompt support! I will continue to monitor the PRs over the next few days
